src/templates/core/category_update.html

```diff
@@ -1,17 +1,47 @@
-{% extends "base/base.html" %}
-
+{% extends "base/base_card.html" %}
 {% load i18n %}
-
-{% block head_title %}{% trans "Update category" %}{% endblock %}
-
-{% block content %}
-
-    <h1>{% trans "Update category" %}</h1>
-
+{% load static %}
+{% load widget_tweaks %}
+{% block head_title %}{% trans "Update category" %}
+{% endblock %}
+{% block card %}
+    <div class="col-lg-12 d-flex justify-content-center mt-3">
+        <h1>{% trans "Update category" %}</h1>
+    </div>
     <form action="{% url "core:category-update" category.id %}" method="post">
         {% csrf_token %}
-        {{ form.as_p }}
-        <button type="submit">{% trans "Update" %}</button>
-    </form>
+        {% for field in form %}
+            <div class="row mb-3 justify-content-center">
+                <div class="col-sm-9 text-secondary">
+                    {{ field }}
+                </div>
+            </div>
+        {% endfor %}
 
+        <div class="row justify-content-center">
+            <div class="col-sm-9 mb-3 text-secondary">
+                <button class="btn btn-primary px-4" type="submit">{% trans "Update" %}</button>
+                <a class="btn btn-outline-danger px-4"
+                   href="{% url 'core:category-list' %}">{% trans "Cancel" %}</a>
+            </div>
+        </div>
+    </form>
 {% endblock %}
+
+{#{% extends "base/base.html" %}#}
+{##}
+{#{% load i18n %}#}
+{##}
+{#{% block head_title %}{% trans "Update category" %}{% endblock %}#}
+{##}
+{#{% block content %}#}
+{##}
+{#    <h1>{% trans "Update category" %}</h1>#}
+{##}
+{#    <form action="{% url "core:category-update" category.id %}" method="post">#}
+{#        {% csrf_token %}#}
+{#        {{ form.as_p }}#}
+{#        <button type="submit">{% trans "Update" %}</button>#}
+{#    </form>#}
+{##}
+{#{% endblock %}#}
```

Comment:
you should remove it or add #TODO row and explain the reasons why you left it here.

Reply:
removed